src/selectors/entities/posts.js

```diff
@@ -260,12 +276,8 @@ export function makeGetPostsAroundPost() {
                     continue;
                 }
 
-                const previousPost = allPosts[slicedPostIds[i + 1]] || {create_at: 0};
-                const formattedPost = formatPostInChannel(post, previousPost, i, allPosts, postsInThread, slicedPostIds, currentUser);
-
-                if (post.id === focusedPostId) {
-                    formattedPost.highlight = true;
-                }
+                const previousPost = allPosts[slicedPostIds[i + 1]] || null;
```

Comment:
@jespino Could you explain logical change here?

Reply:
Sure, the previous post is now A Post or null, not a Post or a custom object to fullfil some requirement elsewhere.

On the other hand, we have the `formatedPost`, generated using `formatPostIChannel` which now receive the `focusedPostId` variable to decide if the `formattedPost` is highlight or not.

Reply:
Do we need to include `|| null` here? I don't know if the difference between null and undefined are meaningful in this code

Reply:
Not really, but I normally prefer an explicit `null` than `undefined` which is normally implicit.